recipe/meta.yaml

```diff
@@ -49,275 +71,224 @@ outputs:
-        - {{ cdt('libselinux-devel') }}      # [linux]
-        - {{ cdt('libxdamage') }}            # [linux]
-        - {{ cdt('libxdamage-devel') }}      # [linux]
-        - {{ cdt('libxfixes') }}             # [linux]
-        - {{ cdt('libxfixes-devel') }}       # [linux]
-        - {{ cdt('libxxf86vm') }}            # [linux]
-        - {{ cdt('libxxf86vm-devel') }}      # [linux]
-        - {{ cdt('libsm-devel') }}           # [linux]
-        - {{ cdt('libice-devel') }}          # [linux]
-        - {{ cdt('libxcb') }}                # [linux]
-        - {{ cdt('libxcb-devel') }}          # [linux]
-        - {{ cdt('expat-devel') }}           # [linux]
-        - {{ cdt('pcre') }}                  # [linux]
-        - {{ cdt('libglvnd-glx') }}          # [linux]
-        - {{ cdt('libglvnd-egl') }}          # [linux]
-        - {{ cdt('libxkbcommon-devel') }}    # [linux]
-        - {{ cdt('xcb-util-devel') }}        # [linux]
-        - {{ cdt('xcb-util-image-devel') }}  # [linux]
-        - {{ cdt('xcb-util-keysyms-devel') }}     # [linux]
-        - {{ cdt('xcb-util-renderutil-devel') }}  # [linux]
-        - {{ cdt('xcb-util-wm-devel') }}          # [linux]
-        - jom                                # [win]
         - python                                 # [build_platform != target_platform]
         - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-        - pyqt-builder                           # [build_platform != target_platform]
-        - sip                                    # [build_platform != target_platform]
-        - make
       host:
         - python
+        - setuptools
+        - wheel
         - pip
-        - sip                                    # [build_platform == target_platform]
-        - toml
-        - pyqt-builder                           # [build_platform == target_platform]
-        - packaging
-        - qt-main
-        - {{ cdt('mesa-libgl-devel') }}          # [linux]
-        - {{ cdt('mesa-libegl-devel') }}         # [linux]
       run:
         - python
-        - {{ pin_subpackage('pyqt5-sip', exact=True) }}
+        - sip
+        - packaging
+        - toml
     test:
-      files:
-        - pyqt_test.py
       imports:
-        - PyQt5
-        - PyQt5.sip
-        - PyQt5.QtCore
-        - PyQt5.QtGui
-        - PyQt5.QtHelp
-        - PyQt5.QtMultimedia
-        - PyQt5.QtMultimediaWidgets
-        - PyQt5.QtNetwork
-        - PyQt5.QtOpenGL
-        - PyQt5.QtPrintSupport
-        - PyQt5.QtQml
-        - PyQt5.QtQuick
-        - PyQt5.QtSvg
-        - PyQt5.QtTest
-        - PyQt5.QtWebChannel
-        - PyQt5.QtWebSockets
-        - PyQt5.QtWidgets
-        - PyQt5.QtXml
-        - PyQt5.QtXmlPatterns
-      commands:
-        - DISPLAY=localhost:1.0 xvfb-run -a bash -c 'python pyqt_test.py'  # [linux and build_platform == target_platform]
-        - pyuic5 --version  # [not win]
+        - PyQt6.sip
```

Comment:
The diff here is very confusing. Please leave the order of outputs as before, to reduce git mis-matching the outputs in the diff.

Reply:
I don't think that I (or the previous author of this diff changed the order). Git might just be confused because of the added output at the top and might be mismatched. Once the temporary` pyqt6-builder` is gone, it should be much easier to compare.

Reply:
You can add the extra output at the end, conda-build will build in order of the dependency graph anyway.

Reply:
Ah, good to know. Thanks!